api-model/src/main/java/io/enmasse/address/model/UnresolvedAddressException.java

```diff
@@ -5,6 +5,9 @@
 package io.enmasse.address.model;
 
 public class UnresolvedAddressException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
```

Comment:
As before - and I can't believe we ever want to serialise an exception object

Reply:
I pushed a commit which switches from adding the serial IDs to simply ignoring the warning.

So we can merge (or revert) this commit.

Reply:
I'm fine with ignoring/suppressing the warning